yb-voyager/src/tgtdb/postgres.go

```diff
@@ -1026,3 +1026,90 @@ func (pg *TargetPostgreSQL) getSchemaList() []string {
 	schemas := strings.Split(pg.tconf.Schema, ",")
 	return schemas
 }
+
+func (pg *TargetPostgreSQL) GetEnabledTriggersAndFks() ([]string, error) {
+	var result []string
+	querySchemaArray := pg.getSchemaList()
+	querySchemaList := strings.Join(querySchemaArray, ",")
+
+	// Check the trigger status using the tgenabled column, which can have three possible values:
+	// 'O' indicates that the trigger is enabled and will fire on the specified events (fully active).
+	// 'D' indicates that the trigger is disabled and will not fire (inactive).
+	// 'R' indicates that the trigger is enabled but will not fire due to certain conditions (e.g., associated with a view).
+	// This query returns only the triggers that are enabled (either 'O' or 'R').
```

Comment:
I don't understand `R` properly. Why won't it be triggered if it's associated with a view? and if it's not going to fire, why should we report it?

Reply:
'R' (replica) status on a trigger means it is enabled but will only fire during replication events, not during regular operations.

Reply:
Also I missed the 'A' status here. Added it now:
'A' (always) status on a trigger in PostgreSQL means that the trigger is enabled and will fire for all operations, regardless of any session replication role settings.